src/components/Template/Template.mdx

```diff
@@ -47,72 +76,271 @@ import {
+        breakpoint
+      </li>
+      <li>Ideal for filtering elements and secondary menus</li>
+    </List>
+  }
+  imageProps={{
+    alt: "Template sidebar region",
+    caption: "Sidebar region",
+    src: "/template/templateSidebarLeft.png",
+  }}
+/>
+}
 
-In order to render a sidebar, the `Template` component needs to be aware of it.
-Pass the location of the sidebar to the `sidebar` prop as either "left" or "right".
+### TemplateMain
 
-For accessibility purposes, you should also structure the DOM logically. This means
-if you are using `sidebar="left"`, place the `TemplateSidebar` before `TemplateMain`.
-If you are using `sidebar="right"`, place the `TemplateSidebar` after.
+{
+
+<DocsWithImage
+  docs={
+    <List type="ul">
+      <li>
+        Positioned below <code>TemplateTop</code> and to either the left or
+        right of
+        <code>TemplateSidebar</code>, when it is rendered
+      </li>
+      <li>
+        May be used in conjunction with <code>TemplateSidebar</code>
+      </li>
+      <li>
+        Spans a perecentage of the central content area when the sidebar is
+        rendered
+      </li>
+      <li>
+        Spans the full width of the central content area when the sidebar is not
+        rendered
+      </li>
+      <li>
+        Content in this region should adhere to the{" "}
+        <Link href="/?path=/docs/style-guide-responsive-grid--docs">
+          responsive grid
+        </Link>{" "}
+        recommendations
+      </li>
+    </List>
+  }
+  imageProps={{
+    alt: "Template main region",
+    caption: "Main region",
+    src: "/template/templateMain.png",
+  }}
+/>
+}
+
+### TemplateMainNarrow
+
+{
+
+<DocsWithImage
+  docs={
+    <List type="ul">
+      <li>
+        Positioned below <code>TemplateTop</code> and to either the left or
+        right of
+        <code>TemplateSidebar</code>, when it is rendered
+      </li>
+      <li>
+        Should not be used in conjunction with <code>TemplateSidebar</code>
+      </li>
+      <li>Contained by the central content area</li>
+      <li>
+        Max width of <code>720px</code>
+      </li>
+      <li>Ideal for displaying content that is primarily text</li>
+    </List>
+  }
+  imageProps={{
+    alt: "Template main narrow region",
+    caption: "Main narrow region",
+    src: "/template/templateMainNarrow.png",
+  }}
+/>
+}
+
+### TemplateBottom
+
+{
+
+<DocsWithImage
+  docs={
+    <List type="ul">
+      <li>
+        Positioned directly below <code>TemplateSidebar</code> and{" "}
+        <code>TemplateMain</code> and above the NYPL universal footer
+      </li>
+      <li>Contained by the central content area</li>
+      <li>Always spans the full width of the central content area</li>
+      <li>
+        Ideal for <code>Banner</code>, <code>FeaturedContent</code>,{" "}
+        <code>NewsletterSignup</code> components
+      </li>
+      <li>Also recommended for custom "announcement" elements</li>
```

Comment:
Some alerts and notifications are already placed on the top of the page. What kind of announcements do you mean?

Reply:
Using the space right before a footer to add announcements or global elements is a common pattern.

Reply:
I will update this statement to the following:
```
Also recommended for persistent or custom "announcement" elements
```